x/incentives/client/rest/rest.go

```diff
@@ -0,0 +1,26 @@
+package rest
+
+import (
+	"github.com/gorilla/mux"
+
+	"github.com/cosmos/cosmos-sdk/client"
+	// this line is used by starport scaffolding # 1
+)
+
+const (
+    MethodGet = "GET"
+)
+
+// RegisterRoutes registers incentives-related REST handlers to a router
+func RegisterRoutes(clientCtx client.Context, r *mux.Router) {
+	// this line is used by starport scaffolding # 2
```

Comment:
What exactly does this refer to?

Reply:
It's for the starport tool:  https://github.com/tendermint/starport

Reply:
Removed these lines that are automatically generated